papergrid/src/lib.rs

```diff
@@ -664,23 +664,48 @@ impl Default for Style {
         Self {
             alignment_h: AlignmentHorizontal::Left,
             alignment_v: AlignmentVertical::Top,
-            indent: Indent {
-                bottom: 0,
-                left: 0,
-                right: 0,
-                top: 0,
-            },
+            padding: Padding::default(),
             span: 1,
         }
     }
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Copy)]
+pub struct Padding {
+    pub top: Indent,
+    pub bottom: Indent,
+    pub left: Indent,
+    pub right: Indent,
+}
+
+impl Default for Padding {
+    fn default() -> Self {
+        Self {
+            top: Indent::default(),
+            bottom: Indent::default(),
+            left: Indent::default(),
+            right: Indent::default(),
+        }
+    }
+}
+
+#[derive(Debug, Clone, Copy)]
 pub struct Indent {
-    pub top: usize,
-    pub bottom: usize,
-    pub left: usize,
-    pub right: usize,
+    pub fill: char,
+    pub size: usize,
+}
```

Comment:
I think creating a `new` method is valuable here?

Reply:
👌🏻 (`new` was added)